prow/jobs_config/knative-extensions/eventing-kafka-broker.yaml

```diff
@@ -1,7 +1,7 @@
 org: knative-extensions
 repo: eventing-kafka-broker
 branches: [main]
-image: us-docker.pkg.dev/knative-tests/images/prow-tests:v20231207-30ee442a2
+image: us-docker.pkg.dev/knative-tests/images/prow-tests:v20231129-81431e1fe
```

Comment:
nit: Even this change will probably be overridden by the next image bumper job, we could stay at `v20231207-30ee442a2` for main already